### ISSUE TITLE
Use Framework or not

### ISSUE THREAD

**Issue by Susmita-Dey** [Susmita-Dey/Codezilla#7]
Need to discuss on this topic whether or not to use any framework for frontend or keep it simple like this.

**Comment by pratikkabade** [comment]
which framework do you want to use?

**Comment by Susmita-Dey** [comment]
@pratikkabade Thinking about React.js or Next.js

**Comment by pratikkabade** [comment]
yes you can do it, afterall what you are doing is just a posting a `http req`

**Comment by Susmita-Dey** [comment]
Yes

**Comment by pratikkabade** [comment]
@Susmita-Dey using libraries will solve your other issues too like #4 #3

**Comment by pratikkabade** [comment]
But i think it's a dead end coz the api takes a long time to get started and openai can terminate it anytime

**Comment by Susmita-Dey** [comment]
@pratikkabade Can it be done without libraries?

**Comment by pratikkabade** [comment]
@Susmita-Dey ofcourse but it's very time consuming considering you have to build everything from scratch

 but why don't you want to use libraries

**Comment by Susmita-Dey** [comment]
@pratikkabade I can use libraries. Let me see
